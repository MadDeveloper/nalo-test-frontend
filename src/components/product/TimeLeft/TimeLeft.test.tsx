import renderer from "react-test-renderer"
import { TimeLeft } from "."

describe("Components > TimeLeft", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<TimeLeft timeLeft="10 hours" />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
