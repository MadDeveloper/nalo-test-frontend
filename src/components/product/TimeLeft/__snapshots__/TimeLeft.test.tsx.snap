// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > TimeLeft should match snapshot 1`] = `
<div
  className="root"
>
  <span
    className="root"
  >
    10 hours
     left ðŸ”¥
  </span>
</div>
`;
